tests/test_default_router.py

```diff
@@ -159,6 +159,10 @@ def test_invalid_field_name(self, template):
             ValueError,
             self.router.add_route, template, {}, ResourceWithId(-1))
 
+    def test_single_character_field_name(self):
+        self.router.add_route(
+            '/search?q={q}', {}, ResourceWithId(22))
```

Comment:
This changeset is simply to prevent this sort of add_route call with a single character parameter from raising an exception.

There is a longer form of this sort of test, but this is what I reached for first. If it is desired, it can be changed: http://stackoverflow.com/a/20275035/1796894